decidim-blogs/config/locales/en.yml

```diff
@@ -1,6 +1,12 @@
 ---
 en:
   activemodel:
+    attributes:
+      post:
+        body: Body
+        decidim_author_id: Author
```

Comment:
This contains also other additional keys compared to the original PR.

Is this on purpose?